llpc/test/shaderdb/general/PipelineVsFs_TestPrimitiveID_Last.pipe

```diff
@@ -0,0 +1,78 @@
+; Test primitiveID when provoking vertex mode is LAST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v --gfxip=10.1.0 %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST: @[[LDS:[^ ]*]] = external addrspace(3) global [{{[0-9]*}} x i32], align 4
+; SHADERTEST: define dllexport amdgpu_gs void @_amdgpu_gs_main(i32
+; SHADERTEST: %{{[0-9]*}} = lshr i32 %{{[^,]*}}, 18
+; SHADERTEST: [[offset:%[0-9]*]] = and i32 %{{[0-9]*}}, 2044
+; SHADERTEST: %{{[0-9]*}} = getelementptr i8, i8 addrspace(3)* bitcast ([{{[0-9]*}} x i32] addrspace(3)* @[[LDS]] to i8 addrspace(3)*), i32 [[offset]]
+; SHADERTEST: [[Addr:%[0-9]*]] = bitcast i8 addrspace(3)* %8 to i32 addrspace(3)*
```

Comment:
What is `%8` in this case?  I guess it is suppose to be the result of the previous instruction.  If so giving it a name that force them to match will make that clearer:

```
; SHADERTEST: [[addr1:%[0-9]*]] = getelementptr i8, i8 addrspace(3)* bitcast ([{{[0-9]*}} x i32] addrspace(3)* @[[LDS]] to i8 addrspace(3)*), i32 [[offset]]
; SHADERTEST: [[Addr2:%[0-9]*]] = bitcast i8 addrspace(3)* [[addr1]] to i32 addrspace(3)*
```